pokepaylib/src/main/java/jp/pokepay/pokepaylib/PartnerAPI/PartnerRequest.java

```diff
@@ -0,0 +1,27 @@
+package jp.pokepay.pokepaylib.PartnerAPI;
+
+import java.util.Map;
+
+import jp.pokepay.pokepaylib.BankAPI.BankRequestError;
+import jp.pokepay.pokepaylib.OAuthAPI.OAuthRequestError;
+import jp.pokepay.pokepaylib.ProcessingError;
+import jp.pokepay.pokepaylib.Request;
+
+public abstract class PartnerRequest {
```

Comment:
When you say "Partner" in pokepay, it is confusing to think of the Partner API.
What about other expressions? (e.g. external)?